Project/src/main/java/app/RegisterController.java

```
@@ -47,10 +48,18 @@ public String newUser(@ModelAttribute User user){
     }
 
     @GetMapping("/")
-    public String defaultPage(Model model) {
+    public String defaultPage() {
         User user = getUser();
-        model.addAttribute("user", user);
-        return "default";
+        switch(user.getRole()) {
+            case EDITOR:
+                return "redirect:/editor";
+            case REVIEWER:
+                return "redirect:/reviewer";
+            case SUBMITTER:
+                return "redirect:/upload";
+            default:
+                return "default";
```

I think this should throw an error code

---

Will make change